pkg/virtual/apiexport/controllers/apireconciler/apiexport_apireconciler_reconcile.go

```diff
@@ -84,18 +86,20 @@ func (c *APIReconciler) reconcile(ctx context.Context, apiExport *apisv1alpha1.A
 		identities[gr] = apiExport.Status.IdentityHash
 	}
 
+	clusterName := logicalcluster.From(apiExport)
+
 	// Find schemas for claimed resources
 	claims := map[schema.GroupResource]*apisv1alpha1.PermissionClaim{}
 	for i := range apiExport.Spec.PermissionClaims {
 		pc := &apiExport.Spec.PermissionClaims[i]
 
-		// APIExport resource trump over claims.
+		// APIExport resources have priority over claimed resources
 		gr := schema.GroupResource{Group: pc.Group, Resource: pc.Resource}
 		if _, found := apiResourceSchemas[gr]; found {
 			if otherClaim, found := claims[gr]; found {
-				klog.Warningf("Permission claim %v for APIExport %s|%s is shadowed by %v", pc, logicalcluster.From(apiExport), apiExport.Name, otherClaim)
+				logger.Info("Permission claim is shadowed by another claim", "claim", pc, "otherClaim", otherClaim)
```

Comment:
`otherClaim` is an object, so use `logging.WithObject()` (but if you already have a claim on the logger from the `pc` object, use `logging.FromPrefix(otherClaim, "otherClaim")`)

Reply:
PermissionClaim is not a full Object

Reply:
ACK